### PR TITLE
[IMP] pylint_odoo: Add check duplicate ids in ir.model.access.csv files

### PR DESCRIPTION
**Pull request by lescobarvx**
[IMP] pylint_odoo: Add check duplicate ids in ir.model.access.csv files

### CODE REVIEW

File: pylint_odoo/checkers/modules_odoo.py [hunk 1]
```diff
@@ -131,6 +136,21 @@ def _check_duplicate_xml_record_id(self):
             return False
         return True
 
+    def _check_duplicate_id_csv(self):
+        '''Check duplicate xml id in ir.model.access.csv files of a odoo module.
```
Comment (pedrobaeza) [inline pylint_odoo/checkers/modules_odoo.py]: Use double quota for docstrings

File: pylint_odoo/misc.py [hunk 2]
```diff
@@ -213,3 +214,13 @@ def get_xml_record_ids(self, xml_file, module=None):
             xml_ids.append(
                 xml_module + '.' + xml_id + '.' + noupdate)
         return xml_ids
+
+    def get_field_csv(self, csv_file, field='id'):
+        '''Get xml ids from csv file
```
Comment (pedrobaeza) [inline pylint_odoo/misc.py]: Double quotes

**Comment by max3903** [comment]
:+1:

**Comment by lescobarvx** [comment]
@pedrobaeza 
The other methods are with single quote, do you want me to make the change by double quote in all methods? I can do it with pleasure

**Comment by pedrobaeza** [comment]
It will be great. This is the PEP 0257 specification: https://www.python.org/dev/peps/pep-0257/:
```
For consistency, always use """triple double quotes""" around docstrings. Use r"""raw triple double quotes""" if you use any backslashes in your docstrings. For Unicode docstrings, use u"""Unicode triple-quoted strings""" 
```

**Comment by moylop260** [comment]
@lescobarvx 
The next check could be the `PEP 0257` :)
This is a python generic check we can add it to [pylint](https://github.com/PyCQA/pylint/) project.

**Comment by lescobarvx** [comment]
@pedrobaeza
pending changes are ready

**Comment by moylop260** [comment]
:+1:

**Comment by pedrobaeza** [comment]
Thanks :+1:

**Pull request merged**